File: backend/app/api/v1/messages.py
from fastapi import APIRouter

router = APIRouter()


@router.get("/{session_id}/messages")
async def list_messages(session_id: str):
    return {"message": f"获取会话 {session_id} 消息列表 - 待实现"}


@router.post("/{session_id}/messages")
async def send_message(session_id: str):
    return {"message": f"发送消息到会话 {session_id} - 待实现"}

File: apps/web/src/api/messages.ts
import { apiFetch } from './client'

export async function fetchMessages(sessionId: string) {
  const res = await apiFetch(`/api/messages/${sessionId}`)
  const data = await res.json()
  return (data as any).items ?? []
}

export async function sendMessage(
  sessionId: string,
  content: string,
  type: 'text' | 'markdown' = 'text',
) {
  const res = await apiFetch(`/api/messages/${sessionId}`, {
    method: 'POST',
    body: JSON.stringify({ content, type, mentions: [] }),
  })
  return res.json()
}
